dcos_installer/upgrades.py

```diff
@@ -0,0 +1,112 @@
+"""Generates a bash script for upgrading by hand """
+
+import gen.installer.util as util
+import gen.template
+from dcos_installer.config import Config
+from dcos_installer.constants import CONFIG_PATH
+from pkgpanda.util import (if_exists, load_json, write_string)
+
+
+node_upgrade_template = """
```

Comment:
I agree, we could take out some of the boilerplate stuff, i am not sure how much value that adds since we dont intend to do upgrades this way in the long run